db/user.go

```diff
@@ -221,3 +222,71 @@ func SyncUsers(course *model.Course, newUsers map[string]*model.User,
 
     return syncResult, nil;
 }
+
+// ResolveUsers is a helper function to maps string representations of roles and * (all roles) to the emails for users with those roles.
+// The function takes an optional course and a list of strings, containing emails, roles, and * as input and returns an alphanumerically sorted slice of lowercase emails without duplicates.
```

Comment:
The list is lexicographically sorted.
You can just say "sorted" or "lexicographically sorted", but "alphanumerically" is incorrect.